src/Veldrid/MTL/MTLGraphicsDevice.cs

```diff
@@ -16,7 +20,7 @@ internal unsafe class MTLGraphicsDevice : GraphicsDevice
         private static readonly Dictionary<IntPtr, MTLGraphicsDevice> s_aotRegisteredBlocks
             = new Dictionary<IntPtr, MTLGraphicsDevice>();
 
-        private readonly MTLDevice _device;
+        private readonly MTLDevice _device;
```

Comment:
There's a lot of unnecessary whitespace changes included in this diff. I can't tell what was actually changed. Can you remove these unexpected diffs from the PR?

Reply:
The previous attempt had much more space changes because editor used `.editorconfig`

Reply:
@mellinoe i can't
![image](https://user-images.githubusercontent.com/36124472/122197259-7147db80-ceb1-11eb-87f6-246930d82ac6.png)
^ this is what i get when trying to edit files on `master` branch.
those space changes are auto fixes by editor